grails-app/services/org/pih/warehouse/core/DocumentService.groovy

```diff
@@ -1635,4 +1635,35 @@ class DocumentService {
         return documents
     }
 
+    void generateAmountOutstandingOnOrdersxcel(OutputStream outputStream, List<Map> data) {
```

Comment:
If you do need to keep this fix the method name to include the E in Excel.

generateAmountOutstandingOnOrdersExcel

Reply:
For now, I removed it. If Kelsey will want this, then I add it back.